<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerspectiveUi</class>
 <widget class="QWidget" name="PerspectiveUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:enabled {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 16px;
color: #FFFFFF;
text-align: center;
}

QPushButton:disabled{
background-color: #7AB7FA;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 16px;
color: #FFFFFF;
text-align: center;
}

QPushButton#pb_back {
background-color: transparent;
border:none;
}

QPushButton#pb_guide {
background-color: transparent;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #0B22C6;
text-align: center;
text-decoration:underline;
}

QFrame#switch_frame {
background-color: transparent;
border-radius: 4px;
border: 1px solid #007AFF;
}

QPushButton#pb_auto, #pb_manual {
background-color: #ffffff;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #007AFF;
text-align: center;
}

Perspective[onManualMode=&quot;true&quot;] QPushButton#pb_manual {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FFFFFF;
text-align: center;
}

Perspective[onManualMode=&quot;false&quot;] QPushButton#pb_auto {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FFFFFF;
text-align: center;
}

</string>
  </property>
  <property name="onManualMode" stdset="0">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="20,640">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="pb_back">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource/resource.qrc">
         <normaloff>:/ui/images/ic_back.png</normaloff>:/ui/images/ic_back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-family: SourceHanSansCN-Medium;
font-size: 18px;
color: #333333;
text-align: right;
}</string>
       </property>
       <property name="text">
        <string>摄像头标定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_body">
     <property name="styleSheet">
      <string notr="true">.QFrame#frame_body {
border-radius: 10px;
background-color: #FFFFFF;
border: 1px solid #E0E0E0;
}

QLabel#lb_text_1,#lb_text_2, #lb_text_3 {
font-family: SourceHanSansCN-Normal;
font-size: 16px;
color: #333333;
text-align: center;
}

QLabel#lb_x, #lb_y, #lb_z {
font-family: SourceHanSansCN-Normal;
font-size: 16px;
color: #000000;
text-align: center;
border: 1px solid #007AFF;
}

QLable#label {
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #333333;
}</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>175</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>175</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QFrame" name="switch_frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>282</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pb_auto">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>自动标定</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_manual">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>手动标定</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="horizontalFrame">
          <property name="minimumSize">
           <size>
            <width>714</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>714</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignRight|Qt::AlignTop">
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>550</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLable#label {
background-color: transparent;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #333333;
}</string>
             </property>
             <property name="text">
              <string>请将摄像头画面中的两条红线与地图矩形框内的中线和底边重合，地图矩形框全部变绿则表明标定成功！</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QPushButton" name="pb_guide">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>操作说明</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">QLabel#label_2
 {
background-color: transparent;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FF0000;
}</string>
            </property>
            <property name="text">
             <string>注意:标定过程中请勿拔出摄像头!</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>59</number>
        </property>
        <property name="rightMargin">
         <number>59</number>
        </property>
        <item>
         <widget class="QLabel" name="lb_live">
          <property name="minimumSize">
           <size>
            <width>592</width>
            <height>444</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>592</width>
            <height>444</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="pb_previous">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>上一步</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="pb_next">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>下一步</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_done">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>完成</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
